test_infra/ess/.stable-snapshot-version

```diff
@@ -0,0 +1 @@
+9.1.0-5affded5-SNAPSHOT
```

Comment:
The JSONs in the buckets contain versions like `9.1.0-5affded5` (without `-SNAPSHOT`). The deployment requires it to end with `-SNAPSHOT`. So, something must attach the `-SNAPSHOT` suffix: either the TF deployment or the UpdateCLI. 
I decided it will be UpdateCLI. The only reason: we keep the version in `.stable-snapshot-version` in the required format